demoproviders/trapdoor/java/org/openyolo/demoprovider/trapdoor/CredentialQueryReceiver.java

```diff
@@ -60,7 +60,7 @@ protected void processCredentialRequest(
         final QueryResponseSender responseSender = new QueryResponseSender(applicationContext);
 
         // Ensure the caller supports the Id and Password authentication method
```

Comment:
The trapdoor provider generally needs an overhaul - it's a hint provider, but was written before we added the hint method. Anyway, fixed the comment and related issues in the meantime until I have time to do more invasive surgery.